contributing/lts-release.md

```diff
@@ -0,0 +1,33 @@
+# Long-Term Stable (lts) Release Channel
+
+The **lts** release channel provides a previous **stable** build for four-week intervals. Approximately every four weeks, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
+
+> Note: The **lts** release channel is available only to AMP Websites; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
+
+## How to Opt-In
+
+### Standard AMP HTML runtime and extension scripts
```

Comment:
maybe add "(recommended for most sites)" to this one?

Reply:
Done